store/util.go

```diff
@@ -0,0 +1,43 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/aau-network-security/haaukins-exercises/model"
+)
+
+var (
+	errorMissingTagName        = fmt.Errorf("error! tag or exercise name is missing")
+	errorMissingInstance       = fmt.Errorf("error! there should be at the least an instance")
+	errorMissingImage          = fmt.Errorf("error! instance image empty")
+	errorMissingFlagConfig     = fmt.Errorf("error! there should be at the least a child in an instance")
+	errorMissingExerciseFields = fmt.Errorf("error! not all the required fields in a child are present")
+)
+
+func checkExerciseFields(ex model.Exercise) error {
```

Comment:
Yeah, needs to be updated, I did not check the other PRs but if it is updated somehow, it would be nice.